### PR TITLE
persisting zip code to event list page

### PR DESCRIPTION
**Pull request by maheshkurma**
persisting zip code to event list page

### CODE REVIEW

**Review by seanlawrenz — COMMENTED** [review]
It looks like we've removed unused variables, but what this feature does isn't immediately clear to me.

File: src/Modules/General/SearchComponent.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ import PlacesAutocomplete, {
 } from "react-places-autocomplete";
 
 const SearchComponent = forwardRef(({ register, errors, onSubmitHandler, searchData}, ref) => {
-  const default_zipcode = (searchData && searchData.zip_code)? searchData.zip_code : "";
+  const default_zipcode = (searchData && searchData.length > 0)? searchData : "";
```
Comment (maheshkurma) [inline src/Modules/General/SearchComponent.js:9]: @seanlawrenz updated from  searchData.zip_code to just searchData
now searchData itself has only zip, but before zip attribute is inside searchData (like searchData.zip_code)

**Comment by sarath-yerra** [comment]
Issues Fixed in this PR:
**Able to Persist Zip on Search Page/Events Page**
Merging it to Master branch.

**Review by sarath-yerra — APPROVED** [review]
Issues Fixed in this PR:
**Able to Persist Zip on Search Page/Events Page**
Merging it to Master branch.

**Pull request merged**